src/test/scala/com.snowplowanalytics.weather/providers/openweather/CacheSpec.scala

```diff
@@ -14,6 +14,8 @@ package com.snowplowanalytics.weather
 package providers.openweather
 
 // scala
+import com.snowplowanalytics.weather.providers.openweather.Errors.InvalidConfigurationError
```

Comment:
you don't need the whole import since you're in the same package and we usually put imports in the same package at the very end of the import list

Reply:
Oh, this is an auto-import from IDE that I missed, gonna fix soon